### PR TITLE
feat(images-plugin): use peer deps

### PR DESCRIPTION
**Pull request by boneskull**
BREAKING CHANGE:

`@appium/images-plugin` now expects to be installed alongside `appium`.

### CODE REVIEW

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* 2.0
  * **PR #16901** <a href="https://app.graphite.dev/github/pr/appium/appium/16901" target="_blank">Graphite</a> 
    * **PR #16902** <a href="https://app.graphite.dev/github/pr/appium/appium/16902" target="_blank">Graphite</a> 
    * **PR #16903** <a href="https://app.graphite.dev/github/pr/appium/appium/16903" target="_blank">Graphite</a> 
    * **PR #16904** <a href="https://app.graphite.dev/github/pr/appium/appium/16904" target="_blank">Graphite</a> 
    * **PR #16905** <a href="https://app.graphite.dev/github/pr/appium/appium/16905" target="_blank">Graphite</a>  (this PR)
    * **PR #16906** <a href="https://app.graphite.dev/github/pr/appium/appium/16906" target="_blank">Graphite</a> 
    * **PR #16907** <a href="https://app.graphite.dev/github/pr/appium/appium/16907" target="_blank">Graphite</a> 
    * **PR #16908** <a href="https://app.graphite.dev/github/pr/appium/appium/16908" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/appium/appium/16905?utm_source=stack-comment).

File: packages/images-plugin/lib/finder.js [hunk 1]
```diff
@@ -359,15 +424,19 @@ export default class ImageElementFinder {
...
+          `screen at ${screenWidth}x${screenHeight}`
+      );
       imgObj = imgObj.resize(screenWidth, screenHeight);
 
       scale.xScale *= (1.0 * screenWidth) / shotWidth;
       scale.yScale *= (1.0 * screenHeight) / shotHeight;
     }
 
-    b64Screenshot = (await imgObj.getBuffer(imageUtil.MIME_PNG)).toString('base64');
+    b64Screenshot = (await imgObj.getBuffer(imageUtil.MIME_PNG)).toString(
```
Comment (mykola-mokhnach) [inline packages/images-plugin/lib/finder.js:437]: it seems the max allowed line length in prettier is too low

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
Should e2e tests pass first?

**Comment by StephanB75** [comment]
Am i allowed to ask here something regarding the greaking change ?
What does it mean that plugins now expects to be installed alongside appium ?
Must they now installed into the appium directory ?

File: packages/images-plugin/lib/finder.js (hunk 1 above)
Comment (boneskull) [reply packages/images-plugin/lib/finder.js:437]: It should be 80 chars.  but it also does extra wrapping.

**Comment by boneskull** [comment]
@StephanB75 If you are using `APPIUM_HOME`, which is typically `~/.appium`, then you don't need to install Appium alongside it.  There's more discussion in #16901

File: packages/images-plugin/lib/finder.js (hunk 1 above)
Comment (mykola-mokhnach) [reply packages/images-plugin/lib/finder.js:437]: in the time of text terminals this probably made sense, but for modern displays I prefer to increase this value to 120 chars

**Comment by boneskull** [comment]
Yes, tests should pass, and if they don't, I messed something up, so I have something to fix here.

**Review by jlipps — APPROVED** [review]

File: packages/images-plugin/lib/finder.js [hunk 2]
```diff
@@ -73,6 +73,11 @@ const DEFAULT_SETTINGS = {
 };
 
 export default class ImageElementFinder {
+  /**
+   *
+   * @param {import('@appium/types').Driver} driver
+   * @param {number} max
```
Comment (jlipps) [inline packages/images-plugin/lib/finder.js:79]: is `max` the name or the description? if the name, it doesn't match, and if description, it's a bit laconic.

**Review by jlipps — APPROVED** [review]

**Pull request merged**